src/components/BInput/BInput.vue

```diff
@@ -0,0 +1,195 @@
+<template>
+  <div class="flex">
+    <label :for="id" :class="{ disabled: disabled }">
+      {{ label }}<span v-if="required" aria-label="Required">*</span>
+    </label>
+    <div v-if="currency" :class="{ currency: true, focused: focused, disabled: disabled, invalid: invalid }" @click="$refs.input.focus()">
+      <span style="font-weight: bold;">$</span>
+      <input v-model="publicValue" v-bind="inputAttributes" v-validate="validations" @focus="focused = true" @blur="focused = false" ref="input" />
+    </div>
+    <input v-else v-model="publicValue" v-bind="inputAttributes" v-validate="validations" :class="{ invalid: invalid }" />
+    <div v-if="invalid" :id="`error-${id}`" class="error">
+      <span v-for="error in errors.all()">
+        {{ error }}
```

Comment:
Will these get separated with commas or do something that makes them all readable?

Reply:
```Turns out it only shows one error at a time. Only once the error is fixed
will it display the next```

Is that actually the experience we want?

Reply:
Have you tried it? It changes immediately (ex. required + email first error says "This is a required field" and then as soon as you start typing it says "Must be a valid email")

Reply:
In this scenario, a11y users and visual users want the same experience. To know what the errors are when they need to fix them. I don't mind only having one show up at a time, but this might be a good question for Steve to answer for our larger UX direction